docs/doc_assets/css/styleguide.scss

```diff
@@ -1506,3 +1408,40 @@ $font-light: 300;
     }
   }
 }
+
+// Demo fixed footer
+.demo-footer {
```

Comment:
It's where the go to back to the site link is located on the demo page.

<img width="1095" alt="screen shot 2016-08-12 at 10 09 04 am" src="https://cloud.githubusercontent.com/assets/5249443/17630615/e4dc7502-6074-11e6-9237-5e5e384507bd.png">